daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/parsers/PState.scala

```diff
@@ -107,12 +112,16 @@ object MPState {
 
 class MPState private () {
 
-  val arrayIndexStack = MStackOfLong()
+  val arrayIterationIndexStack = MStackOfLong()
+  val occursIndexStack = MStackOfLong()
+
+  def moveOverOneArrayIterationIndexOnly() =
+    arrayIterationIndexStack.push(arrayIterationIndexStack.pop + 1)
 
-  def moveOverOneArrayIndexOnly() = arrayIndexStack.push(arrayIndexStack.pop + 1)
-  def moveBackOneArrayIndexOnly() = arrayIndexStack.push(arrayIndexStack.pop - 1)
+  def arrayPos = arrayIterationIndexStack.top
```

Comment:
Can this be renamed `arrayIterationPos`?

Reply:
Thanks for finding that one. I should have done a grep for arrayPos.